packages/firestore/package.json

```diff
@@ -44,7 +44,8 @@
     "test:node:persistence:prod": "node ./scripts/run-tests.js --main=index.node.ts --persistence 'test/{,!(browser|lite)/**/}*.test.ts'",
     "test:travis": "ts-node --compiler-options='{\"module\":\"commonjs\"}' ../../scripts/emulator-testing/firestore-test-runner.ts",
     "test:minified": "(cd ../../integration/firestore ; yarn test)",
-    "api-report": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' ts-node ./scripts/api-report.ts",
+    "api-report": "yarn api-report:api-json && TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' ts-node ./scripts/api-report.ts",
+    "api-report:api-json": "rm -rf temp && api-extractor run --local --verbose",
```

Comment:
Does the rollup build have to happen first? CI is complaining it can't find the mainEntryPointFilePath d.ts file which I think is generated during the build, but the api-report command here is run during `prebuild`?

Reply:
Oh good catch. `api-report:api-json` should run after `ts-node ./scripts/api-report.ts` which creates the pruned d.ts files.